content/software-security/cves/cve-why-care.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "Why Care About Software Vulnerabilities?"
+description: "An overview of software vulnerability impacts and the significance of CVEs in vulnerability management practices"
+lead: "An overview of software vulnerability impacts and the significance of CVEs in vulnerability management practices"
+type: "article"
+date: 2023-07-13T19:46:58+00:00
+lastmod: 2023-07-13T19:46:58+00:00
+contributors: ["Michelle McAveety"]
+draft: false
+tags: ["Overview"]
+images: []
+menu:
+  docs:
+    parent: "cves"
+weight: 002
+toc: true
+---
+
+Software products are prone to [*vulnerabilities*](/software-security/cves/cve-intro/) which, if exploited by an attacker, may negatively impact the systems and consumers relying on them. Attacks against vulnerable software systems can result in the unintended exposure and misuse of sensitive data (like the theft of user account credentials). In other cases, these attacks could affect the provision of a service or compromise critical infrastructure that relies on the software. Given the considerable threat that they can pose, it is important that developers spend time mitigating vulnerabilities to protect against hackers seeking to exploit them.
+
+Addressing the vulnerabilities present in your software helps secure the systems you support, use, and maintain. In this article, you will explore why you should care about vulnerabilities as a software developer, and learn about federal regulations that draw importance to CVE management (such as FedRAMP).
+
+
+## As a Developer
+
+Discovering and mitigating software vulnerabilities is a difficult – but necessary – task for developers to tackle. Development teams are ultimately the authorities which are able to remediate vulnerabilities at the source. Depending on the severity of a vulnerability, its exploitation could significantly impact the integrity of the software product, as well as the trust, safety, and operations of the consumers who rely on it.
+
+Vulnerability *triage* is an important step developers must take to work toward reducing risks posed by vulnerabilities in their software. Using [*vulnerability scanners*](https://owasp.org/www-community/Vulnerability_Scanning_Tools) can give developers information about the number, type, and severity of CVEs present in their work. With this data, developers can prioritize critical CVEs first, therefore ensuring that major security concerns are addressed first.
+
+
+## Federal Regulations
+
+With notable software supply chain security attacks occurring in recent years (such as the [SolarWinds attack in 2020](/software-security/glossary/#solarwinds-hack)), the U.S. federal government has increased efforts to improve U.S. cybersecurity. These initiatives aim to strengthen software supply chain security by promoting safe development habits, such as frequent vulnerability scanning.
```

Comment:
"Safe" and "secure" can only ever be "safer" and "more secure" as there is nothing 100% unfortunately 😄 

```suggestion
With notable software supply chain security attacks occurring in recent years (such as the [SolarWinds attack in 2020](/software-security/glossary/#solarwinds-hack)), the U.S. federal government has increased efforts to improve U.S. cybersecurity. These initiatives aim to strengthen software supply chain security by promoting safer development habits, such as frequent vulnerability scanning.
```

Reply:
Makes sense, thank you for pointing that out!